main/main_camera.gd

```diff
@@ -14,7 +14,7 @@ func _unhandled_input(event: InputEvent) -> void:
 	if is_current():  
 		if event is InputEventMouseMotion:
 			if event.button_mask == MOUSE_BUTTON_MASK_LEFT:
-				position -= event.relative
+				position -= event.relative * zoom
```

Comment:
When zooming in the zoom value increases and when zooming out, the zoom value decreases.
It should be `position -= event.relative / zoom`. Multiplying made it even worse.

Reply:
Good call